MEvoLibGUI/pyproject.toml

```diff
@@ -32,29 +30,31 @@ keywords = [
     "GUI",
     "MEvoLib",
 ]
-[tool.setuptools]
-py-modules = []
-
 dependencies = [
     "celery >= 5.3.6",
     "Django >= 4.2.7",
-    "MEvoLib >= 2.0",
+    "MEvoLib @ git+https://github.com/JAlvarezJarreta/mevolib",
     "nextflow >= 22.10.7",
     "nextflowpy >= 0.8.1",
+    "python-dotenv >= 1.0.1",
     "redis >= 5.0.1",
-    "python-dotenv >= 1.0.1"
 ]
 
 [project.optional-dependencies]
 dev = [
     "black",
+    "mypy",
     "pylint",
-    "python-dotenv"
+    "python-dotenv",
+    "types-requests",
 ]
 
 [project.urls]
 repository = "https://github.com/nachorequeno/mevolib-ui"
 
+[tool.setuptools]
```

Comment:
We have to talk to Nacho about this, as he told me it makes his installation to crash, so I am unsure of keeping it or not.